lib/Parse/ParseDecl.cpp

```diff
@@ -6748,6 +6748,15 @@ Parser::parseDeclVar(ParseDeclOptions Flags,
     // If we syntactically match the second decl-var production, with a
     // var-get-set clause, parse the var-get-set clause.
     if (Tok.is(tok::l_brace)) {
+
+      //  give more useful errors for forget "do". see
```

Comment:
This comment might be more helpful if you could describe how the code here "give[s] more useful errors" in this circumstance. Also, please note that the style in this codebase is to use one space (not two) after `//` and to capitalize the first letter of each sentence.